packages/entities/entities-redis-configurations/src/partials-endpoints.ts

```diff
@@ -0,0 +1,31 @@
+const konnectBaseApiUrl = '/v2/control-planes/{controlPlaneId}/core-entities'
+const KMBaseApiUrl = '/{workspace}'
+
+export default {
+  list: {
+    konnect: {
+      all: `${konnectBaseApiUrl}/partials`,
+    },
```

Comment:
Shall we remove `all` in this file?
```suggestion
    konnect: `${konnectBaseApiUrl}/partials`,
```

We distinguish `all` and `forEntity` endpoints for entities that can be scoped to another entity, e.g. routes or plugins: https://github.com/Kong/public-ui-components/blob/d3c535205e133480952e71bc1885e39b2a4e7ee7/packages/entities/entities-plugins/src/plugins-endpoints.ts#L7-L8. Redis configuration cannot be scoped (correct me if I'm wrong), so we don't need `all` here, just like certificates: https://github.com/Kong/public-ui-components/blob/d3c535205e133480952e71bc1885e39b2a4e7ee7/packages/entities/entities-certificates/src/certificates-endpoints.ts#L6

Reply:
updated